README.md

```diff
@@ -75,7 +75,19 @@ const Map = ReactMapboxGl({
 ## Why are `zoom`, `bearing` and `pitch` Arrays ?
 If those properties changed at the mapbox-gl-js level and you don't update the value kept in your state, it will be unsynced with the current viewport. At some point you might want to update the viewport value (zoom, pitch or bearing) with the ones in your state but using value equality is not enough. Taking zoom as example, you will still have the unsynced zoom value therefore we can't tell if you want to update the prop or not. In order to explicitly update the current viewport values you can instead break the references of those props and reliably update the current viewport with the one you have in your state to be synced again.
 
-## [Version 3.0 Documentation](docs/API.md)
+
+## [Version 4.0 Documentation](docs/API.md)
```

Comment:
I would not add breaking changes in the readme, can you just add the breaking changes in the description of the PR and I will take everything from every PR descriptions and write some release notes when publishing v4

Reply:
Alright, I took those changes out. The only things changed in the README now are the parts from the example that use `style` and `containerStyle`. I also updated the PR description to include the breaking changes